django/db/models/functions.py

```diff
@@ -47,7 +47,8 @@ def as_sqlite(self, compiler, connection):
         return super(ConcatPair, self).as_sql(compiler, connection)
 
     def as_mysql(self, compiler, connection):
-        self.coalesce()
+        self.function = 'CONCAT_WS'
+        self.template = '%(function)s("", %(expressions)s)'
```

Comment:
Good point, it should be using single quotes. It supports both for string literals until you turn on ANSI_QUOTES sql_mode, which I don't have locally but others might.